docs/data-sources/space_by_path.md

```diff
@@ -24,6 +26,11 @@ data "spacelift_space_by_path" "space" {
 output "space_description" {
   value = data.spacelift_space_by_path.space.description
 }
+
+// The following example shows how to use a relative path. If this is used in a stack in the root space, this is identical to using a path of `root/second space/my space`.
```

Comment:
Same here - I think you need to update the example tf file rather than the markdown file and regenerate the docs.